index.html

```diff
@@ -1,88 +1,61 @@
-        }
-
-        /* .center {
-            width: 70%;
-            margin: auto;
-            border: 1px solid #4287f5;
-
-        } */
-
-        h1,
-        h4 {
-            text-align: center;
-        }
-
-        #rightAlign {
-            text-align: right;
-            font-weight: bold;
-        }
-    </style>
-</head>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
 
-<body>
+  <body>
     <center>
-        <div class="center">
-            <h1>Vaccine Availability Status</h1>
-            <h4 id="refreshTimerText"></h4>
-            <table class="input">
-                <tbody>
-                    <tr>
-                        <td>Date [DD-MM-YYYY]</td>
-                        <td><input type="text" id="dateForCheckingAvailability" /></td>
-
-                    </tr>
-
-                    <tr>
-                        <td>Disctrict Code(s) [Comma Separated]</td>
-                        <td><input type="text" id="disctrictCodes" /></td>
-                        </td>
-                    </tr>
-                    <tr>
-                        <td id="rightAlign">Press Enter to Search:</td>
-                        <td>
-                            <center> <button type="button" onclick="search();">Search</button> </center>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-            <p>
-                Options</br>
-                <input type="checkbox" id="optionsAge18plus" checked>Age 18+ &amp &lt 45 </input></br>
-                <input type="checkbox" id="optionsAge45plus">Age 45+ </input> </br>
-                <input type="checkbox" id="optionsDose1" checked>Dose 1 </input></br>
-                <input type="checkbox" id="optionsDose2">Dose 2 </input>
-            </p>
-            <h4 id="API_CALL_Count"></h4>
-
-
-            <table class="output" id="mainTable">
-                <tbody>
-                    <tr>
-                        <td>District Code</td>
-                        <td>District Name</td>
-                        <td>Availability</td>
-                        <td>Name</td>
-                        <td>Pin Code</td>
-                        <td>Available Capacity for Dose 1</td>
-                        <td>Available Capacity for Dose 2</td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
+      <div class="center">
+        <h1>Vaccine Availability Status</h1>
+        <h4 id="refreshTimerText"></h4>
+        <table class="input">
+          <tbody>
+            <tr>
+              <td>Date [DD-MM-YYYY]</td>
+              <td><input type="text" id="dateForCheckingAvailability" /></td>
+            </tr>
+
+            <tr>
+              <td>Disctrict Code</td>
+              <td><input type="text" id="districtCodes" value="392, 395" /></td>
+            </tr>
+          </tbody>
+        </table>
+        <button type="button" id="btn" onclick="main()">Search</button>
```

Comment:
on click we are calling main() function rather than search() function.
However, there is no "main()" function defined in app.js file.
Did this work while testing the website locally?

Reply:
Okay I realized that there are 2 JS files here.
Legacy app.js and newer script.js.
Would still like the onClick call function to be same on both scripts so when we want to switch to either of the scripts, we can just do so by changing the "src" on <script> element at the end of body